supplementary_style_guide/glossary_terms_conventions/general_conventions/p.adoc

```diff
@@ -228,6 +228,22 @@ When combined with privileges, permissions define a user's or resource's overall
 
 *See also*:
 
+[[playbook]]
+==== image:images/yes.png[yes] playbook (noun)
+*Description*: Playbooks are Ansible's configuration, deployment, and orchestration language.
```

Comment:
```suggestion
*Description*: Playbooks are the configuration, deployment, and orchestration language for Ansible Automation Platform.
```
- Per ISG, "Do not use 's to show the possessive form of brand names or product names" - https://www.ibm.com/docs/en/ibm-style?topic=references-product-service-names#general-guidelines"